projects/storefrontlib/src/lib/product/components/product-details/product-images/product-images.component.scss

```diff
@@ -1,96 +1,33 @@
-$cx-product-images-pdp-items-small-lastchild-trbl-margin: 0 0 0 0 !default;
-$cx-product-images-pdp-mobile-small-trbl-margin: 0 15px 0 0 !default;
-$cx-product-images-pdp-mobile-small-height: 80px !default;
-$cx-product-images-pdp-mobile-small-width: 80px !default;
-.cx-product-images {
-  &__item-image {
-    width: $cx-product-images-pdp-item-image-width;
-  }
-  &__items {
-    text-align: center;
-    margin: $cx-product-images-pdp-items-trbl-margin;
-  }
-  &__item-zoom--icon {
-    @include var-color('color', $cx-product-images-pdp-zoom-color);
-    border: $cx-product-images-pdp-zoom-border;
-    border-radius: $cx-product-images-pdp-zoom-border-radius;
-    display: block;
-    width: $cx-product-images-pdp-zoom-width;
-    height: $cx-product-images-pdp-zoom-height;
-    float: right;
-    margin: $cx-product-images-pdp-zoom-trbl-margin;
-    cursor: pointer;
-    &:after {
-      content: '+';
-      bottom: 13px;
-      position: relative;
-      left: 3px;
-      font-size: 29px;
-    }
-    &:hover {
-      @include var-color('color', $cx-product-images-pdp-zoom-color-hover);
-    }
-  }
-  &__item--small {
-    margin: $cx-product-images-pdp-small-trbl-margin;
-    border: $cx-product-images-pdp-small-border;
-    @include var-color(
-      'border-color',
-      $cx-product-images-pdp-small-border-color
-    );
-    width: $cx-product-images-pdp-small-width;
-    height: $cx-product-images-pdp-small-height;
-    display: inline-block;
-    &:hover {
-      @include var-color(
-        'border-color',
-        $cx-product-images-pdp-small-border-color-hover
-      );
-    }
-    &:last-child {
-      margin: $cx-product-images-pdp-items-small-lastchild-trbl-margin;
-    }
-    @include media-breakpoint-down(sm) {
-      width: $cx-product-images-pdp-mobile-small-width;
-      height: $cx-product-images-pdp-mobile-small-height;
-      margin: $cx-product-images-pdp-mobile-small-trbl-margin;
+:host {
+  display: flex;
+  flex-direction: var(--cx-flex-direction, column);
+
+  --cx-border-color: var(--cx-light);
+  --cx-border-width: 0.1vw;
+}
+
+.thumbs {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+
+  cx-picture {
+    transition: all var(--cx-transition-time);
+
+    &:not(.active) {
+      cursor: pointer;
     }
-    img {
-      width: $cx-product-images-pdp-small-img-width;
-      height: $cx-product-images-pdp-small-img-height;
-      @include var-color(
-        'border-color',
-        $cx-product-images-pdp-small-img-border-color
-      );
-      display: inline-block;
-      padding: $cx-product-images-pdp-small-trbl-padding;
-      @include media-breakpoint-down(sm) {
-        width: $cx-product-images-pdp-mobile-small-width;
-        height: $cx-product-images-pdp-mobile-small-height;
-      }
+    overflow: hidden;
+    position: relative;
+
+    border: var(--cx-border-width) var(--cx-border-color) var(--cx-border-style);
+
+    margin: var(--cx-margin, 0.5vw);
```

Comment:
This is based on the pretext that there won't be a global value for --cx-margin which makes sense as it would be weird to have the same margin (& padding) throughout the app. What I'm trying to say is that perhaps we should document somewhere what vars are safe to apply "globally".

Reply:
Yes, we will document this as soon as we have agreed as a group.